examples/wandb_sweeps/make_wandb_sweep.py

```diff
@@ -0,0 +1,98 @@
+"""Creates a hyperparameter sweep in the wandb API.
+
+Manually modify functions below with different Dicts to change the hyperparameters, 
+or sweep method (defaults to random).
+
+See here for details: https://docs.wandb.ai/guides/sweeps/define-sweep-configuration"""
+
+import wandb
+
+import argparse
+from typing import Dict
+
+
+def get_optimization_params() -> Dict:
```

Comment:
I would suggest that this and the following function just be replaced with a `PARAMS = ` constant dictionary. I am not seeing much benefit for them as a function.